package.json

```diff
@@ -63,14 +63,17 @@
     "immutable": "^3.8.1",
     "imports-loader": "^0.6.5",
     "normalize.css": "^4.2.0",
-    "react": "^15.2.1",
-    "react-dom": "^15.2.1",
+    "react": "^15.3.0",
+    "react-dom": "^15.3.0",
     "react-redux": "^4.4.5",
     "react-router": "^2.6.1",
     "react-router-redux": "^4.0.5",
+    "react-tap-event-plugin": "^1.0.0",
```

Comment:
https://github.com/zilverline/react-tap-event-plugin

For touch devices. You've probably heard of iOS's dreaded 300ms tap delay. React's onClick attribute falls prey to it. Facebook's working on a solution in the form of TapEventPlugin, but it won't be made available until 1.0.
https://developers.google.com/web/updates/2013/12/300ms-tap-delay-gone-away